scout/server/blueprints/panels/templates/panels/panels.html

```diff
@@ -27,6 +28,48 @@
   </div>
 {% endblock %} <!-- end of content_main -->
 
+
+
+ 
+<!-- Create a Bootstrap card containing text bar and submit button for searching -->
+{% macro search_gene_card() %}
+<div class="container-float">
+  <div class="card mt-3">
+    <div class="card-body">
+      <form action="{{ url_for('panels.panels') }}" enctype="multipart/form-data" method="POST">
```

Comment:
enctype="multipart/form-data"  is generally used if the form contains a file upload. Since this is not the case I think it would be best to omit the enctype so it will be the default (application/x-www-form-urlencoded).

Reply:
Removed!